ext/xmlreader/php_xmlreader.stub.php

```diff
@@ -74,8 +74,8 @@ public function setRelaxNGSchema(?string $filename) {}
     public function setRelaxNGSchemaSource(?string $source) {}
 
     /** @return bool|XMLReader */
-    public static function XML(string $source, ?string $encoding = null, int $options = 0) {}
+    public static function XML(string $source, ?string $encoding = null, int $flags = 0) {}
```

Comment:
On the topic of `$sqlite3`, should the first parameter be called `$XML`? Can we maybe use `$source` in sqlite3 as well?

Reply:
I'm not sure I understand the connection to sqlite3. `$source` here is in the sense of "source code" not in the sense of "source and target". Where would we be able to use `$source` in the sqlite API?

Reply:
I was talking about the `backup()` method. The first parameter could be just `$destination` same as we have just `$source` here. I didn't mean the parameters are the same in these two extensions, just we should follow the same pattern. 
The first parameter in `xml()` is actually an XML string, but we call it source, could we not call the first parameter in `sqlite3::backup()` just `$destination` to follow a similar pattern? Sorry for my confusing way of writing.

Reply:
I'm a bit confused as well. Are you thinking of the name `$source` as "where it gets the data from"? That the `$sqlite3` parameter in `sqlite3::backup()` and the `$source` parameter in `xmlreader::xml()` are related in that way? 

Object instances like the `$sqlite3` parameter are often named after the class that it is an instance of. Is that the source (I'm sorry) of your confusion?

Reply:
No, I am not making a functional connection between these two extensions. I am purely talking about name choice. I.e. why have chosen `$source` instead of `$xml`? 

Following the same logic, we can use the name `$destination` instead of `$sqlite3`. I know the type of the parameter is different, but I am only talking about naming convention.

Reply:
> why have chosen $source instead of $xml?

Because it is already clear from the context that it is expecting XML. That means we can use a name that is more generic and consistent with other extensions.

You do have a fair argument when you say that `$sqlite3` should be renamed to something more generic, like `$destination`, for the same reason. It has been suggested to do so, but general consensus appears to be that the name should reflect the type of object in these cases.